ui/src/app/shared/edge/config.2018.7.ts

```diff
@@ -76,10 +77,23 @@ export class ConfigImpl_2018_7 extends ConfigImpl implements DefaultTypes.Config
         let simulatorDevices: string[] = [];
         let evcsDevices: string[] = [];
         let thresholdDevices: string[] = [];
+        let essType: string[] = [];

```

Comment:
Instead of storing the essType here, you could also store the maxApparentPower here directly. It would save you from the extra 'if' in CurrentDataAndSummary.